### PR TITLE
Add database tests to explain performance.

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

File: lib/restrictions.js [hunk 1]
```diff
@@ -149,15 +160,25 @@ export async function update({restriction} = {}) {
  *
  * @param {object} options - Options to use.
  * @param {string} options.id - The ID of the restriction.
+ * @param {boolean} [options.explain] - An optional explain boolean.
```
Comment (dlongley) [inline lib/restrictions.js:163]:
```suggestion
 * @param {boolean} [options.explain=false] - An optional explain boolean.
```

File: lib/restrictions.js [hunk 2]
```diff
@@ -179,17 +200,25 @@ export async function get({id} = {}) {
  *   restriction applies to.
  * @param {string} options.resource - The ID of the resource that the
  *   restriction applies to.
+ * @param {boolean} [options.explain] - An optional explain boolean.
```
Comment (dlongley) [inline lib/restrictions.js:203]:
```suggestion
 * @param {boolean} [options.explain=false] - An optional explain boolean.
```

File: test/test.js [hunk 3]
```diff
@@ -36,20 +34,4 @@ bedrock.events.on('bedrock.init', async () => {
   handlers.setUseHandler({handler: ({meter} = {}) => ({meter})});
 });
 
-bedrock.events.on('bedrock.ready', async () => {
```
Comment (dlongley) [inline test/test.js:39]: Why is it ok to remove all this stuff?

@gannan08, can you confirm that this shouldn't be here?

**Review by dlongley — APPROVED** [review]
Thanks! LGTM, but I'd like to get @gannan08's sign off here since some of the code he added to get compatibility with meters is being removed. This may be fine (it may no longer be needed due to other updates that simplified what tests need), but we need to confirm with him.

File: test/test.js (hunk 3 above)
Comment (JoshDunnDev) [reply test/test.js:39]: This was what was suggested by @gannan08 to remove. I am sure he can confirm however the test suite would not run without this code removed due to the new meter changes.

**Review by gannan08 — APPROVED** [review]

**Pull request merged**